### PR TITLE
Implement VitsSVC resume training / finetune feature

### PR DESCRIPTION
**Pull request by viewfinder-annn**
**✨ Description:** 

This PR addresses bugs in VitsSVC module, mainly related to:
1. implement resume training / finetune feature
2. fix bug for multiple acoustic content training

**🚧 Related Issues:** 

[#91 ](https://github.com/open-mmlab/Amphion/issues/91)

**👨‍💻 Changes Proposed:** 

- [x] Fix bug in `models/svc/vits/vits.py`, removed useless content_vec feature dependency.
- [x] Refractor VitsSVCTrainer to detach from TTSTrainer

**🧑‍🤝‍🧑 Who can review?** 

@RMSnow @lmxue

**🛠 TODO**

- [x] Further clean up config
- [x] Refractor VitsSVCTrainer to detach from TTSTrainer

**✅ Checklist:**

- [ ]  Code has been reviewed
- [x]  Code complies with the project's code standards and best practices
- [x]  Code has passed all tests
- [x]  Code does not affect the normal use of existing features
- [x]  Code has been commented properly
- [ ]  Documentation has been updated (if applicable)

### CODE REVIEW

**Review by RMSnow — CHANGES_REQUESTED** [review]
1. Implement VitsSVCTrainer from the SVC's trainer as the base instead of the TTS ones
2. Refine the config json files. Move some common setting to a `config/svc.json`, and make the `exp_config.json` as user-friendly as possible

**Comment by viewfinder-annn** [comment]
Just fixed problem 1, now this PR doesn't affect tasks other than SVC.

**Comment by viewfinder-annn** [comment]
Now the default training data config is the same across DiffWaveNet and VitsSVC. The `exp_config.json` is now user-friendly with fewer exposed options. I'll refractor SVC preprocess to `svc.json` in another PR soon.

File: models/svc/vits/vits.py [hunk 1]
```diff
@@ -240,24 +238,23 @@ def forward(self, data):
     @torch.no_grad()
     def infer(self, data, noise_scale=0.35, seed=52468):
         # c, f0, uv, g
-        c = data["contentvec_feat"].transpose(1, 2)
         f0 = data["frame_pitch"]
         g = data["spk_id"]
 
-        if c.device == torch.device("cuda"):
+        if f0.device == torch.device("cuda"):
```
Comment (RMSnow) [inline models/svc/vits/vits.py:244]: How about if the input features don't contain F0? If you need to get the frame length, use the `target_len`. I'm not sure about how to get the specific device, maybe you can follow [this line](https://github.com/open-mmlab/Amphion/blob/4e8e7a5a5975420413b1f6fd338757314d882741/models/svc/diffusion/diffusion_trainer.py#L65C41-L65C41).

**Review by RMSnow — CHANGES_REQUESTED** [review]

**Review by lmxue — APPROVED** [review]
LGTM.

File: models/svc/vits/vits.py (hunk 1 above)
Comment (viewfinder-annn) [reply models/svc/vits/vits.py:244]: Thanks! I'll detach the data dependency from f0 in the future. While f0 is always a  necessity since default generator needs it(nsf-hifigan).

**Review by RMSnow — APPROVED** [review]
